tests-common/src/main/java/org/jboss/weld/test/util/Utils.java

```diff
@@ -177,12 +183,19 @@ public static boolean isProxy(Object proxy) {
     private static class TCCLObjectInputStream extends ObjectInputStream {
 
         private final ClassLoader classLoader;
+        private ClassLoader lastResortClassLoader = null;
```

Comment:
`lastResortClassLoader`? Choose a better name pls :D The original `classLoader` could be renamed to `tccl` or something like that.Also the field should be final.